src/code/z_actor.c

```diff
@@ -1766,13 +1753,13 @@ s32 func_8002F9EC(GlobalContext* globalCtx, Actor* actor, CollisionPoly* poly, s
     return false;
 }
 
-// Local data used for Farore's Wind light (stored in BSS, possibly a struct?)
-LightInfo D_8015BC00;
-LightNode* D_8015BC10;
-s32 D_8015BC14;
-f32 D_8015BC18;
+// Local data used for Farore's Wind light
+static LightInfo D_8015BC00;
+static LightNode* D_8015BC10;
+static s32 D_8015BC14;
+static f32 D_8015BC18;
 
-void func_8002FA60(GlobalContext* globalCtx) {
+void Actor_InitFaroresWindPointer(GlobalContext* globalCtx) {
```

Comment:
I'm not a fan of the pointer suffix here. What do you mean with this? Like, a C pointer or something in-game?

Reply:
It means like the literal pointer that shows the player where the spell was casted. Maybe something like `Actor_InitFaroresWindSpellIndicator`, I'm not sure, still doesn't sound great.

Reply:
I think that name is better but I agree it doesn't sound great either.

Reply:
Note:
Should be renamed to `Actor_InitFaroresWindMarker`